README.md

```diff
@@ -38,9 +38,13 @@ It's also needed to extend the installer-path section:
         "type:npm-asset"
     ],
 ```
+And add a new installer-types section next to it:
```

Comment:
This could be made a bit clearer. maybe explain, that it should be added under the "extra" key.